library/build.gradle

```diff
@@ -113,3 +115,25 @@ dependencies {
 }
 
 apply from: rootProject.file('gradle/gradle-mvn-push.gradle')
+
+task aarSize {
+  doLast {
+    File aar = file("build/outputs/aar/mapzen-android-sdk-release.aar")
+    println ""
+    println "AAR Size: ${aar.length()} bytes"
```

Comment:
This is great although we do still need a way to calculate the total size with transitive dependencies so when someone asks "How much bigger will the Mapzen SDK make my app?" we can give a more accurate answer.